Client Support/Renewal/2020-09-02/System/itemInfo_EN.lua

```diff
@@ -43452,7 +43450,9 @@ tbl = {
 			"________________________",
 			"^0000CCType:^000000 Accessory",
 			"^0000CCWeight:^000000 10",
-			"^0000CCArmor Level:^000000 1",
			"________________________",
			"^0000CCRequirement:^000000",
```

Comment:
Just funny how my local file is fine regarding that, but the uploaded one not xD.